### ISSUE TITLE
File system error does not been ignored when using `--skip-on-error` flag

### ISSUE THREAD

**Issue by moontai0724** [hareku/fanbox-dl#60]
Using Command with `--skip-on-error`

```
.\fanbox-dl.exe --sessid ... --skip-on-error --creator (someone)
```

But still got terminated:

```
time=2024-12-22T22:08:21.624+08:00 level=INFO msg="Launching Pixiv FANBOX Downloader!" version=0.25.0 commit=276f7bb9936a6989a324caa60f8accdc35d24ab3 date=2024-10-31T04:32:30Z
time=2024-12-22T22:08:21.640+08:00 level=INFO msg="Start downloading" creator_id=(someone)
time=2024-12-22T22:08:22.875+08:00 level=INFO msg=Downloading order=0 assetType=image title=うちよそ（commission）
time=2024-12-22T22:08:23.604+08:00 level=INFO msg=Downloading order=1 assetType=image title=うちよそ（commission）
time=2024-12-22T22:08:24.385+08:00 level=ERROR msg="fanbox-dl Error" error="failed downloading of \"hikigaeru\": check whether does image exist: stat file: CreateFile images\\(someone)\\2023-12-26-10月末くらいに自分用に描いたついででイラコ ンに応募して「もしやあればいいな」くらいに思いつつ結構真面目に期待しちゃってたんだけど結果として箸にも棒にも掛からない結果に終わったやつ（作品自体は気に入ってます）-0-xMVrzgz8gMvzVxUCy8j4B1ZU.png: The filename, directory name, or volume label syntax is incorrect."
time=2024-12-22T22:08:24.386+08:00 level=ERROR msg="The error log seems a bug, please open an issue on GitHub" url=https://github.com/hareku/fanbox-dl/issues
```

It seems the file path is too long, but the error does not being catched.

---

Version: 0.27.1 / 0.25.0
Execution Platform: Windows 11
File Storage: Using SMB connect to Linux based Synology NAS

**Comment by hareku** [comment]
Thank you. This is fixed at `v0.27.2`.

**Comment by moontai0724** [comment]
So fast! It works like a charm now! Thank you!
